plugin/clone/src/clone_common.cc

```diff
@@ -98,16 +81,121 @@ int Ha_clone_common_cbk::synchronize_engines() {
   // Size reverse-engineered from ER_CLONE_SERVER_TRACE used by log_error. No
   // way to track it automatically, but very unlikely it would silently shrink.
   char msg_buf[512];
-  snprintf(msg_buf, sizeof(msg_buf), "engine positions: %.*s",
+  snprintf(msg_buf, sizeof(msg_buf), "%s: %.*s", object_name.data(),
```

Comment:
To printf-format an `std::string_view`, you always have to use sized formatting: `%.s*`, similar to `json_str_buf` in the same call here. The reason is that string views do not have to be nul-terminated